src/main/java/org/parabot/core/ui/BotUI.java

```diff
@@ -64,11 +65,34 @@ public static BotUI getInstance() {
         return instance;
     }
 
+    public static boolean deleteDirectory(File directory) {
```

Comment:
Would probably be better to call the static function of [Directories#clearCache](https://github.com/Parabot/Parabot/blob/master/src/main/java/org/parabot/core/Directories.java#L276) directly.